docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -57,6 +57,10 @@ Set the systemd unit name prefix for pods. The default is *pod*.
 
 Set the systemd unit name separator between the name/id of a container/pod and the prefix. The default is *-*.
 
+#### **--template**
+
+Add template specifiers to run multiple services from the systemd unit file. Applicable only when --new is set.
```

Comment:
We should also mention that it only works on containers but not on pods.

Reply:
DONE :+1: